secureli/actions/action.py

```diff
@@ -235,8 +247,9 @@ def _run_post_install_scan(
 
         if secret_test_id := metadata.security_hook_id:
             self.action_deps.echo.print(
-                f"{config.languages} supports secrets detection; running {secret_test_id}."
+                f"The following language(s) support secrets detection: {format_sentence_list(config.languages)}"
             )
+            self.action_deps.echo.print("running {secret_test_id}.")
```

Comment:
It seems weird to me that we mention languages "supporting" secrets detection. `detect-secrets` I believe is language agnostic. I dug a bit into the code and it looks like we actually have the capability to specify different secrets-detection engines for different languages, but I'm not quite sure when that would be useful. (`detect-secrets` is associated with the `base` language, which is inherited by all languages, therefore every language will support it.)

I'm not sure if we need any changes here, but I wouldn't be against tweaking the verbiage here or removing the output that specific languages support secrets detection. (We could also change that line to a debug statement since if we think there's some value for developers of seCureLI.)

```suggestion
            )
            self.action_deps.echo.print("Checking for added secrets with \"{secret_test_id}\".")
```

Reply:
I've thought the same thing previously, and perhaps the implementation for actually determining if languages support secrets detection needs to be updated. IMO, It's good for the user to know that secrets are being checked for each language on the off chance one of them doesn't. I think the implementation could change to actually log languages that support secret detection since we just happen to use the same engine for each at the moment.